test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -390,8 +390,8 @@ class nntrainer_BatchNormalizationLayer
 
   virtual int reinitialize(bool _last_layer = false) {
     int status = super::reinitialize(_last_layer);
-    // loadFile("test_5_BNLayerInput.in", in);
-    // loadFile("test_5_BNLayerWeights.in", layer);
+    loadFile("test_5_BNLayerInput.in", in);
+    loadFile("test_5_BNLayerWeights.in", layer);
```

Comment:
I think this name was changed to tc_bn_1_BNLayerInput.in & tc_bn_1_BNLayerWeights.in.